openkfw/TruBudget
# 1728 add new organization and node

### Checklist

### Description
- add new form for adding new organization with node address with basic Yup validation and Formik
- rework css within Requests for new organizations so we can have clean list of requests which can be approved or declined
<img width="1055" alt="Screenshot 2024-03-22 at 14 13 18" src="https://github.com/openkfw/TruBudget/assets/55734106/656c36c4-fbd7-4d69-9fde-f569d364bc23">

### How to test

1. Go to Hamburger menu and click on Nodes
2. Click on Submit button within Add new organization Card, validation errors will pop up because fields cannot stay empty
3. Fill in Organization name and Node address and click Submit button
4. New request for organization register should pop inside Requests for new organizations

Closes #1728 

Closes ModifyMe
